src/test/shared/ui/input.test.ts

```diff
@@ -0,0 +1,281 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as assert from 'assert'
+import * as vscode from 'vscode'
+import * as input from '../../../shared/ui/input'
+
+describe('createInputBox', async () => {
+    let testInput: vscode.InputBox | undefined
+
+    afterEach(() => {
+        if (testInput) {
+            testInput.dispose()
```

Comment:
Should we also set `testInput` to undefined?

Reply:
I haven't seen a situation where this has been necessary, but I will add it for completeness